package.json

```diff
@@ -12,15 +12,15 @@
   "engines": {
     "node": ">=4"
   },
-  "yscripts": {
+  "xscripts": {
     "_comment1": "use this for the scripts when using yalc",
     "_comment2": "change name of this object to 'scripts' and the current scripts to say 'xscripts'",
     "_comment3": "and revert when done.",
     "prepublishOnly": "rm -fr ./dist & babel ./src --out-dir ./dist -s inline && rm -rf ./dist/demos ./dist/__tests__",
     "xprepublishOnly": "rm -fr ./dist & babel ./src --out-dir ./dist -s inline"
   },
   "scripts": {
-    "prepublishOnly": "rm -fr ./dist & babel ./src --out-dir ./dist -s inline && yarn test:unit",
+    "prepublishOnly": "rm -fr ./dist & babel ./src --out-dir ./dist -s inline",
```

Comment:
Why remove the unit tests?

Reply:
Robert left a lot of things combined that interfere with my way of working. I even use my own scripts section when I work on this package. Placing the (failing) tests as part of "prepublish" made it so I could actually test the change I was making locally. 

I'd rather not keep the unit tests up to date. I consider this package to be on life support only. So I had to prevent them from being executed so that publishing works.

(note: I have already published this to NPM, since publishing works from your local files, not github; I have tested this both apps that use it, namely tCC and gA)